<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edycja_urzadzenie</class>
 <widget class="QDialog" name="edycja_urzadzenie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>331</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Załaduj tabelę</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="u_model">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>195</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wpisz nazwisko&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_IMIEa_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>131</y>
     <width>133</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Marka</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_NAZWISKOa_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>176</y>
     <width>133</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Model</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="u_marka">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wpisz imię&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj nowe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edytuj...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
